tests/test_dse.py

```diff
@@ -162,7 +162,7 @@ def test_tti_rewrite_aggressive_wmpi():
 
 @switchconfig(profiling='advanced')
 @pytest.mark.parametrize('kernel,space_order,expected', [
-    ('centered', 8, 168), ('centered', 16, 300)
+    ('centered', 8, 148), ('centered', 16, 264)
```

Comment:
What happened there?

Reply:
a consequence of now having 3 time-invariant temporaries , instaed of 4. Also a ~10% flop reduction, consistent across all space orders